Analysis/ingrid/likelihood.nim

```diff
@@ -1265,6 +1279,13 @@ proc filterClustersByVetoes(h5f: var H5File, h5fout: var H5File,
         centerY = h5f[(chipGroup / "centerY"), float64]
         rmsTrans = h5f[(chipGroup / "rmsTransverse"), float64]
         evNumbers = h5f[(chipGroup / "eventNumber"), int64].asType(int)
+      case fileInfo.timepix
+      of Timepix1:
+        discard
+      of Timepix3:
+        if ctx.vetoCfg.useToACut or ctx.vetoCfg.useToAlnLCut:
+          let toaLength = h5f[(chipGroup / "toaLength"), int64].asType(int)
```

Comment:
```suggestion
          toaLength = h5f[(chipGroup / "toaLength"), int64].asType(int)
```
If you keep the `let`, it shadows the `toaLength` variable defined in the outer scope in the scope of the `case` and `if`. So effectively you never write to the actual `toaLength` variable you care about.